### PR TITLE
feat: remove env config in public key api and cert rotation api

### PR DESCRIPTION
**Pull request by capri-xiyue**
remove env config in public key api and cert rotation api

### CODE REVIEW

File: cmd/cert-rotation/main.go [hunk 1]
```diff
@@ -82,16 +90,29 @@ func realMain(ctx context.Context) error {
...
-	defer cleanup.GracefulClose(logger, kmsClient)
+	defer util.GracefulClose(logger, kmsClient)
 
-	config, err := config.LoadCryptoConfig(ctx, []byte{})
+	cryptoCfg, err := config.LoadCryptoConfig(ctx, []byte{})
 	if err != nil {
 		return fmt.Errorf("failed to load config: %w", err)
 	}
 
+	firestoreDoc, err := util.ParseFirestoreDocResource(cryptoCfg.FirestoreDocResourceName)
```
Comment (sethvargo) [inline cmd/cert-rotation/main.go:100]: I don't know if we want this to be firestore per the doc just shared out. I'm worried about the coupling here.

Comment (yolocs) [reply cmd/cert-rotation/main.go:100]: If we're unsure about firestore, maybe we should defer using firestore at all. It's not like we're missing functionality without it. Thoughts?

Comment (sethvargo) [reply cmd/cert-rotation/main.go:100]: Agree on deferring

Comment (capri-xiyue) [reply cmd/cert-rotation/main.go:100]: Ok. I will revert the previous PR which already uses firestore https://github.com/abcxyz/jvs/pull/91

**Comment by capri-xiyue** [comment]
closed because we are deferring the firestore

**Pull request closed without merging**